application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/WolfImage.java

```diff
@@ -0,0 +1,52 @@
+package org.togetherjava.tjbot.commands.mathcommands.wolframalpha;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+
+@JsonRootName("img")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@SuppressWarnings("unused")
+final class WolfImage {
```

Comment:
odd class name. `WolframAlphaImage` or just `Image` (due to package context)

Reply:
I think that is a very long name, I was thinking of refactoring the class name, package name and test class name anyway. The command is `wolf` too.

Reply:
In my opinion it shouldnt be `wolf` but `wolfram-alpha`. I think it also was `wolfram-alpha` in my initial proposal.
`wolf` is (imo) misleading.

Reply:
well the initial command was `=wolf` in Mathbot

Reply:
Resolved as well